### DISCUSSION TITLE
GSOC 2024 Project

### DISCUSSION THREAD

**Discussion by AkintolaRichard** [sosy-lab/benchexec#1008] [General]
Hi @PhilippWendler,

I am interested in implementing the Timestamps in logs Project of the GSOC 2024, I saw this information ```BenchExec is a benchmarking framework for Linux (written in Python)```, and I must say that this aligns with my skills, and also with me being a Linux user.
I am really excited and looking forward to working on the project under your guidance. I would love to get directions from you on how I can go about contributing, although I have gotten the ```paperwork on benchmarking```.

Thank you for having me,
Richard Akintola

**Comment by PhilippWendler** [comment]
Hi and welcome!

for getting to know BenchExec the best way is to try it out and to have a look at the paper. Of course we also have [documentation](https://github.com/sosy-lab/benchexec/blob/main/doc/INDEX.md). I don't think there is anything else in general that you would need to start with.

If you have concrete questions just ask me.

Greetings
Philipp

**Reply by AkintolaRichard** [reply]
Thank you for the warm welcome, I will reach out to you as I progress in checking the book out.

**Comment by AkintolaRichard** [comment]
Hello @PhilippWendler,
I have chekcked out the paper, and currently
I am on the documentation, but I am facing an issue with setting up benchexec. After following the suggested workaround solution, the issue still persists. Attached is the image containing the error message.

![benchmark_error](https://github.com/sosy-lab/benchexec/assets/74435323/a07f581f-53eb-4e3f-8376-b528e26b46fc)

**Reply by AkintolaRichard** [reply]
I have found way around the aforementioned issue, I got a new one, below is the screenshot.

![runexec_error](https://github.com/sosy-lab/benchexec/assets/74435323/518c75ea-8948-40ee-86c6-e65e9ccc2434)

**Reply by PhilippWendler** [reply]
Great! The remaining error is https://github.com/sosy-lab/benchexec/issues/776

**Reply by AkintolaRichard** [reply]
Thank you, it worked perfectly.

**Comment by AkintolaRichard** [comment]
Hi @PhilippWendler ,
I did intend to take up the Timestamp in logs project, but I won't be able to take up the project anymore due to changes made on my school calendar which will prevent me from giving the project the required commitment and I feel sorry about this. Peradventure it wasn't taken up during GSoC, I will take it up after my school exams which is actually my final exam in school.

Thank you for your supports, I do appreciate them.
Richard Akintola

**Reply by PhilippWendler** [reply]
Thank you for reporting back, and good luck with your school exams!
